Doc/Tutorial/chapter_align.tex

```diff
@@ -158,10 +158,9 @@ \subsection{Single Alignments}
 \noindent Note the website should have an option about showing gaps as periods (dots) or dashes, we've shown dashes above.  Assuming you download and save this as file ``PF05371\_seed.faa'' then you can load it with almost exactly the same code:
 
 \begin{minted}{python}
```

Comment:
You need to change the format from python to pycon if you want to add the leading ``>>>`` but why do this at all?

If you turned them into testable doctests, I could understand. But as documentation only, I would find them much easier to use without the ``>>>`` for copy-and-paste.

Reply:
Python itself uses `>>>` in its documentation. Personally I find it clearer because it shows what is Python code and what is not.

Reply:
It does help, true, although in the current PDF output I find the colour formatting is enough. The HTML Tutorial will look much better once we move to RST and Sphinx in place of LaTeX.